eng/CredScanSuppression.json

```diff
@@ -8,6 +8,10 @@
     {
       "file": [ "sdk/attestation/azure-security-attestation/test/ut/crypto_test_collateral.cpp" ],
       "_justification": "File contains a PEM encoded RSA private key used by test code."
+    },
+    {
+      "file": [ "sdk/identity/azure-identity/test/perf/inc/azure/identity/test/externals/perf.combo.pem" ],
+      "_justification": "File contains a PEM encoded RSA private key used by perf test code."
```

Comment:
What is that private key used for? How do we know that it's not the key to some existing Azure resource?

Reply:
No , this certificate is not used for no live services, it is used just to generate the recordings that i will add later on. when the creds are initialized i need to pass in a valid cert file that gets validated during the construction of the instance. thus the need for a valid one.

Reply:
To be clear: I'm looking for some kind of documentation in the code which will explain why this private key isn't a security vulnerability.

Right now, all that I see is that the certificate is used in authentication, and that's very concerning.